src/payment.js

```diff
@@ -384,11 +357,11 @@ Payment.prebuild = function (absoluteFee) {
 
     // compute max spendable limits per each fee-per-kb
     var maxSpendablesPerFeePerKb = function (e) {
-      var c = Transaction.filterUsableCoins(payment.coins, e.fee);
-      var s = Transaction.maxAvailableAmount(c, e.fee);
+      var c = Transaction.filterUsableCoins(payment.coins, e[1] * 1000);
+      var s = Transaction.maxAvailableAmount(c, e[1] * 1000);
       return s.amount;
     };
-    payment.maxSpendableAmounts = payment.fees.estimate.map(maxSpendablesPerFeePerKb);
+    payment.maxSpendableAmounts = Object.entries(payment.fees).map(maxSpendablesPerFeePerKb);
```

Comment:
I don't think `Object.entries` is standard, does babel take care of that? Although, if these arrays are changed to objects, please use [`R.mapObjectIndexed`](http://ramdajs.com/docs/#mapObjIndexed) instead.